app/models/camaleon_cms/category.rb

```diff
@@ -30,21 +33,21 @@ def post_type
   end
 
   private
+
   def set_site
-    pt = self.post_type
-    self.site_id = pt.site_id unless self.site_id.present?
-    self.status = pt.id unless self.status.present?
+    pt = post_type
+    site_id = pt.site_id unless site_id.present?
```

Comment:
The `site_id.present?` may now well reference the local variable `site_id` and not `self.site_id`. Check for the same issue elsewhere in the diff including the following line.